patreon/schemas/campaign.py

```diff
@@ -0,0 +1,67 @@
+class Attributes(object):
```

Comment:
first thought: deploying a new schema on patreon.com kicks off a script which opens a PR against this repo. seemed wayyy out of scope for this PR, but definitely something to think about.